apps/meteor/app/lib/server/startup/settings.ts

```diff
@@ -1334,6 +1291,24 @@ settingsRegistry.addGroup('Message', function () {
 		public: true,
 	});
 
+	// TODO: deprecate this setting in favor of App
+	this.add('API_EmbedDisabledFor', '', {
```

Comment:
This you can remove, and please re add the others on the same line so the diff will not highlight as changes

Reply:
removed